Sources/Confidence/EventSenderEngine.swift

```diff
@@ -106,6 +109,10 @@ final class EventSenderEngineImpl: EventSenderEngine {
         )
     }
 
+    func flush() {
+        writeReqChannel.send(manualFlushEvent)
```

Comment:
Could this be an alternative?
```
        self.uploadReqChannel.send(EventSenderEngineImpl.sendSignalName)
        self.flushPolicies.forEach { policy in policy.reset() }
```
So we don't need to treat the manual flush as a policy

Reply:
I updated the PR description with some explanation about this.